web-marketplace/src/pages/CreditClassDetails/CreditClassDetails.tsx

```diff
@@ -63,6 +63,14 @@ function CreditClassDetails({
   const [isSellFlowStarted, setIsSellFlowStarted] = useState(false);
   const setConnectWalletModal = useSetAtom(connectWalletModalAtom);
 
+  const { data: buyModalOptionsContent } = useQuery(
+    getBuyModalOptionsQuery({ sanityClient: client }),
+  );
+  const buyModalOptions = buyModalOptionsContent?.allBuyModalOptions[0];
```

Comment:
Here we are assuming that the first card is the schedule call link. We are also making this assumption in `BuySellOrderFlow`.